go/chat/convsource.go

```diff
@@ -165,7 +166,7 @@ func (s *baseConversationSource) postProcessThread(ctx context.Context, uid greg
 
 func (s *baseConversationSource) TransformSupersedes(ctx context.Context,
 	unboxInfo types.UnboxConversationInfo, uid gregor1.UID, msgs []chat1.MessageUnboxed) ([]chat1.MessageUnboxed, error) {
-	transform := newBasicSupersedesTransform(s.G())
+	transform := newBasicSupersedesTransform(s.G(), false)
```

Comment:
on any of these calls when you pass unnamed value in to call this can you add `/* useDeletePlaceholders */` so it's easier to read in the future?